--- a/tools/clang/scripts/build.py
+++ b/tools/clang/scripts/build.py
@@ -1079,6 +1079,9 @@
     bootstrap_targets = 'X86'
     if sys.platform == 'darwin':
       # Need ARM and AArch64 for building the ios clang_rt.
       bootstrap_targets += ';ARM;AArch64'
+    if sys.platform.startswith('linux') and platform.machine() == 'aarch64':
+      # Need AArch64 for native ARM64 Linux builds
+      bootstrap_targets += ';AArch64'
     bootstrap_args = base_cmake_args + compiler_wrapper_cmake_args + [
         '-DLLVM_TARGETS_TO_BUILD=' + bootstrap_targets,
